L1Trigger/L1CaloTrigger/src/ParametricCalibration.cc

```diff
@@ -0,0 +1 @@
+#include "L1Trigger/L1CaloTrigger/interface/ParametricCalibration.h"
```

Comment:
any multi-line function implementations from the header file should be moved here

Reply:
done